atest/testdata/output/listener_interface/body_items_v3/tests.robot

```diff
@@ -59,6 +59,19 @@ WHILE
         BREAK
     END
 
+WHILE with modified limit
+    [Documentation]    FAIL 
+    ...    WHILE loop was aborted because it did not finish within the limit of 1 iterations. Use the 'limit' argument to increase or remove the limit if needed.
```

Comment:
This failure spec confused me, because it gives an impression that the test would fail with this message and in that case modifications wouldn't work. It turned out that the expected test status is set explicitly when this test is validated on `atest/robot` side, That means that this spec isn't used for anything and can be removed. I can do that myself and I'll consider changing how other tests handle specifying their expected status as well.